frame/nomination-pools/fuzzer/src/call.rs

```diff
@@ -224,6 +224,7 @@ fn main() {
 		MaxPoolMembers::<T>::set(Some(10_000));
 		MaxPoolMembersPerPool::<T>::set(Some(1000));
 		MaxPools::<T>::set(Some(1_000));
+		GlobalMaxCommission::<T>::set(Some(Perbill::from_percent(90)));
```

Comment:
```suggestion
		GlobalMaxCommission::<T>::set(Some(Perbill::zero()));
```

I recall the fuzzer expects no commission.